storage_service/locations/models/event.py

```diff
@@ -142,6 +155,11 @@ class File(models.Model):
         help_text="Accession ID of originating transfer")
     origin = UUIDField(editable=False, unique=False, version=4, blank=True,
         help_text="Unique identifier of originating Archivematica dashboard")
+    normalized = models.BooleanField(default=None,
+        help_text="Whether or not file has been normalized")
+    validated = models.BooleanField(default=None,
```

Comment:
What's the difference between `False` and `None` for `normalized` and `validated`? Should they default to `False`?